apptests/validator.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+
+from youtube_transcript_api import YouTubeTranscriptApi
+
+import models
+
+
+class ModelClassValidator:
+    def __init__(self, class_name, language, model_type):
+        module_attrs = vars(models)
+        model_class = module_attrs.get(class_name)
+        self.model_instance = model_class()
+        self.language = language
+        self.model_type = model_type
+
+    def get_yt_transcript(self, video_id: str) -> str:
+        transcripts = YouTubeTranscriptApi.list_transcripts(video_id)
+
+        srt = transcripts.find_generated_transcript(language_codes=["en"]).fetch()
+
+        return " ".join(fragment["text"] for fragment in srt)
```

Comment:
Provide a transcription of the ground truth `sample.mp3` as a text file in the `apptests` folder instead of downloading it.